### ISSUE TITLE
cluster health does not print out json

### ISSUE THREAD

**Issue by ajs11174** [etcd-io/etcd#10778]
I am just running a simple endpoint health command and expecting the output to be json
````bash
root@ip-10-0-0-10:/home/ubuntu# ETCDCTL_API=3 etcdctl --cacert /etc/etcd/pki/ca.crt \
--cert /etc/etcd/pki/client.crt --key /etc/etcd/pki/client.key \
--endpoints "https://10.0.0.10:2379" endpoint --cluster health --write-out json

https://10.0.0.10:2379 is healthy: successfully committed proposal: took = 17.043735ms
https://10.0.8.10:2379 is healthy: successfully committed proposal: took = 2.440481ms
https://10.0.4.10:2379 is healthy: successfully committed proposal: took = 1.686717ms
`````
Here is my etcdctl/etcd version
````bash
root@ip-10-0-0-10:/home/ubuntu# ETCDCTL_API=3 etcdctl --cacert /etc/etcd/pki/ca.crt \
--cert /etc/etcd/pki/client.crt --key /etc/etcd/pki/client.key version

etcdctl version: 3.3.13
API version: 3.3
````

Other commands do print out correctly. For example cluster status prints out just fine
````bash
ETCDCTL_API=3 etcdctl --cacert /etc/etcd/pki/ca.crt --cert /etc/etcd/pki/client.crt --key /etc/etcd/pki/client.key --endpoints "https://m0.etcd.dealcube:2379" endpoint --cluster status --write-out json
[{"Endpoint":"https://10.0.4.10:2379","Status":{"header":{"cluster_id":16453928603903837334,"member_id":280618240456672560,"revision":431256,"raft_term":177},"version":"3.3.13","dbSize":10039296,"leader":280618240456672560,"raftIndex":540029,"raftTerm":177}},{"Endpoint":"https://10.0.0.10:2379","Status":{"header":{"cluster_id":16453928603903837334,"member_id":3546093130203668075,"revision":431256,"raft_term":177},"version":"3.3.13","dbSize":10117120,"leader":280618240456672560,"raftIndex":540029,"raftTerm":177}},{"Endpoint":"https://10.0.8.10:2379","Status":{"header":{"cluster_id":16453928603903837334,"member_id":11471290683035990776,"revision":431256,"raft_term":177},"version":"3.3.13","dbSize":9900032,"leader":280618240456672560,"raftIndex":540029,"raftTerm":177}}]
````

**Comment by jingyih** [comment]
I did a quick search, 3.3 does not support json output format. The support for "--write-out" is added in master branch: https://github.com/etcd-io/etcd/commit/d019d3141a469f1d0b06ffa4a50bc1b248d4f50d.

**Comment by jingyih** [comment]
@gyuho Do we want to backport https://github.com/etcd-io/etcd/commit/d019d3141a469f1d0b06ffa4a50bc1b248d4f50d?

**Comment by gyuho** [comment]
@jingyih Sure, can you help? Thanks.

**Comment by jingyih** [comment]
I will backport this to 3.1, 3.2 and 3.3.

**Comment by jingyih** [comment]
Fixed by #10782.
